pkg/events/events.go

```diff
@@ -5975,6 +5976,36 @@ var Definitions = eventDefinitions{
 					{MapName: "prog_array", MapIndexes: []uint32{tailKernelWrite}, ProgName: "trace_ret_kernel_write_tail"},
 					{MapName: "prog_array", MapIndexes: []uint32{tailSendBin}, ProgName: "send_bin"},
 				},
+				Events: []eventDependency{
+					{VfsWrite},
+					{VfsWritev},
+					{KernelWrite},
```

Comment:
These events overload the system and it might be that the user will choose to only capture, without having these events.
Although I know they are needed for the "written_files" to work - we need to reconsider adding them as a dependancy.